problems/arrayMethods.js

```diff
@@ -64,15 +98,23 @@ const numbersOnly = () => {};
  * @returns {boolean} Whether or not all elements are the same.
  */
 
-const isAllSame = () => {};
+const isAllSame = (items) => {
+  return items.every((el) => {
+    return el === items[0];  
+  });
+};
 
 /**
  * Takes in an array and returns whether or not all elements in the array are odd.
  * @param {number[]} nums
  * @returns {boolean} All odd?
  */
 
-const isAllOdd = () => {};
+const isAllOdd = (nums) => {
+  return nums.every((el) => {
+    return el % 2 === 1;
```

Comment:
This is great.

You can also write "return el % 2" because it's either going to return 0, a falsey, or 1 a truthy